src/App.svelte

```diff
@@ -72,7 +72,7 @@
     <script
       defer
       data-domain="app.radicle.xyz"
-      src="https://plausible.io/js/plausible.js"></script>
+      src={`${base}plausible.js`}></script>
```

Comment:
Will this even work if we serve it from our own domain? I'm not familiar with how plausible works, but I wonder if it's not a requirement that it gets pulled from their servers.

Their docs even mention proxying instead of bundling their script: https://plausible.io/docs/proxy/introduction.

Reply:
Hmm good catch 🤔 
My idea was to avoid fetching the js file, which delayed the load completion of the site.
Maybe we could remove the script altogether and use the npm package that doesn't rely on downloading the script?
https://github.com/plausible/plausible-tracker wdyt?

Reply:
Yeah, the npm package sounds good.

Reply:
b1cb59a